csharp/nuget/Facebook.Yoga.nuspec

```diff
@@ -0,0 +1,46 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Nuget spec for the C# wrapper and its associated native dlls... This is uses the sqllite style packaging...